pkgs/utf8proc.sh

```diff
@@ -0,0 +1,59 @@
+# ================================================================
+# Compile utf8proc
+# ================================================================
+
+[ -e $STAGE/utf8proc ] && ( set -xe
+    cd $SCRATCH
+
+    . "$ROOT_DIR/pkgs/utils/git/version.sh" JuliaStrings/utf8proc,v
+    until git clone --depth 1 --single-branch -b "$GIT_TAG" "$GIT_REPO"; do echo 'Retrying'; done
+    cd utf8proc
+    
+    # ------------------------------------------------------------
+
+    . "$ROOT_DIR/pkgs/utils/fpm/pre_build.sh"
+
+    (
+        case "$DISTRO_ID" in
+        'centos' | 'fedora' | 'rhel')
+            set +xe
+            . scl_source enable devtoolset-8
+            set -xe
+            export CC="gcc" CXX="g++"
+            ;;
+        'ubuntu')
+            export CC="gcc-8" CXX="g++-8"
+            ;;
+        esac
+
+        . "$ROOT_DIR/pkgs/utils/fpm/toolchain.sh"
+
+        mkdir -p build
+        cd $_
+
+        cmake                                       \
```

Comment:
Please read utf8proc's CMakeLists.txt and see what option should you use.
For example, should we enable `UTF8PROC_ENABLE_TESTING` and run the unit tests?

Reply:
utf8proc will run test if UTF8PROC_ENABLE_TESTING is enabled